app/assets/javascripts/admin/util.js.erb

```diff
@@ -1,6 +1,6 @@
 $(document).ready(function() {
   $('.datetimepicker').datetimepicker({
-    dateFormat: Spree.translations.date_picker,
+    dateFormat: "yy-mm-dd",
```

Comment:
It looks like the default has already been set to "yy-mm-dd" for `en.yml`, which is correct. It's only on other languages where the key isn't explicitly set in the locale file that it defaults to "yy/mm/dd", and then breaks. I think ignoring the locale settings for this one is the best option, as alternate settings will break site the functionality.